src/Illuminate/Http/Resources/Json/ResourceResponse.php

```diff
@@ -102,10 +102,14 @@ protected function haveAdditionalInformationAndDataIsUnwrapped($data, $with, $ad
     /**
      * Get the default data wrapper for the resource.
      *
-     * @return string
+     * @return string|null
```

Comment:
This function would return `null` before this PR already in cases where `::withoutWrapping()` was used